router/batchrouter/batchrouter.go

```diff
@@ -673,10 +673,12 @@ func (brt *HandleT) copyJobsToStorage(provider string, batchJobs *BatchJobsT, is
 		if isDestInterrupted {
 			if _, ok = interruptedEventsMap[eventID]; !ok {
 				eventsFound = true
+				totalBytes += len(string(job.EventPayload) + "\n")
```

Comment:
Since we are doing `string(job.EventPayload) + "\n"` twice. This will be created twice?